### DISCUSSION TITLE
Contenidos examen.

### DISCUSSION THREAD

**Discussion by elbosco** [IIC2133-PUC/2024-2#191] [Preguntas Materia] [answered]
Buenas tardes,
Quería preguntar si para el examen entrara todo el contenido del semestre o habrá puntos específicos donde la prueba tendrá un enfoque, por ejemplo, en arboles. 
Saludos y gracias de antemano,

**Comment by IgnacioHqz** [comment]
También quisiera sumarme a la pregunta y consultar sobre el resumen. Se podrá realizar de dos hojas o debe ser de una? Muchas gracias!

**Comment by tomastrivino** [comment, marked as answer]
Hola,
Les recomiendo estar atentos a esa información en Canvas, ya que aún no está definida.
Saludos!

@CataAlamos

**Comment by JTUnger** [comment]
@tomastrivino El profesor nos comento que quizás no todas las secciones alcanzaban a pasar todo y lo que estaba en duda es Bellman-Ford, se ha discutido si entra esta materia?
